RyanNoelk/OpenEats
# Feature/ingredient groups

The feature adds ingredient groups and sub recipes. Ingredient Groups allow users to group there ingredients into groups for better sounds a visibility. Sub-recipes allows users to use other recipes as ingredients.

To Test: 
- Migrate your database: `python manage.py migrate`.
- Create/Modify a recipe and add groups or sub-recipes.
- Confirm your changes worked as expeced.
